app/views/content_items/show.html.erb

```diff
@@ -30,7 +30,9 @@
     </tr>
     <tr>
       <td>Organisation</td>
-      <td><%= @organisation.title %></td>
+      <td>
+        <%= list_comma_separated_words @content_item.organisations.collect(&:title) %>
```

Comment:
What about something like:
```ruby
<%= render stringify_organisations(@content_item.organisations) %>
```

I find this approach more expressive, and the view is less dependent of implementation details.
What are your thoughts?